manifests/alertmanager.pp

```diff
@@ -107,6 +107,7 @@
   Array $templates,
   String[1] $user,
   String[1] $version,
+  Array $mute_time_intervals,
```

Comment:
can you please add some unit tests for the change, add puppet-strings documentation at the top of the file and enforce the datatype for the array elements?